.env.example

```
@@ -20,3 +20,18 @@ JWT_SECRET_KEY=supersecretkey
 ENVIRONMENT=local
 
+# --- Utterance segmentation (api/services/segmentation.py) ---
+# All optional; defaults shown. Uploaded media is decoded to 16 kHz mono and
+# split into utterance WAVs using Silero VAD before entering the labeling queue.
+# SEGMENTER_BACKEND=silero              # silero | energy (energy = adaptive RMS fallback)
+# SEGMENTER_VAD_THRESHOLD=0.5           # speech probability trigger (0-1)
+# SEGMENTER_MIN_SPEECH_MS=250           # ignore speech bursts shorter than this
+# SEGMENTER_MIN_SILENCE_MS=300          # silence needed to end a speech region
+# SEGMENTER_SPEECH_PAD_MS=150           # padding kept around each speech region
+# SEGMENTER_MIN_UTTERANCE_S=1.0         # drop utterances shorter than this
+SEGMENTER_MAX_UTTERANCE_S=35.0        # split utterances longer than this
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)